README.md

```diff
@@ -1572,6 +1573,33 @@ Cluster membership updated in the ProxySQL database!
 ```
 The weights corresponding to the node `192.168.56.33:3306` in the writer-config hostgroups has been updated to the new value `1111`. Note that only writer-config hostgroup has been updated since the node doesn't have a corresponding entry in the writer hostgroup.
 
+### Extra options
```

Comment:
Not sure of the need for Extra Options in the documentation. 
The descriptions are close to the current set of descriptions in the Options table. There are no examples or extra details.